src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx

```diff
@@ -21,8 +22,8 @@ const maxDelay = 2000;
 describe("AttributeDropdown", () => {
     function renderComponent(props: any = {}) {
         const {
-            projectId = "storybook",
-            onApply = (f: (...params: any[]) => any /* TODO: make the types more specific (FET-282) */) => f,
+            projectId = "projectId",
```

Comment:
Why the changes in the project id?

Reply:
Because we used projectId everywhere in the links, so to be consistent with it I changed it.

Reply:
Ok :)